vpr/src/route/route_timing.cpp

```diff
@@ -1354,7 +1359,7 @@ std::vector<t_heap> timing_driven_find_all_shortest_paths_from_route_tree(t_rt_n
                                                                           RouterStats& router_stats) {
     //Add the route tree to the heap with no specific target node
     int target_node = OPEN;
-    auto router_lookahead = make_router_lookahead(e_router_lookahead::NO_OP);
+    auto router_lookahead = make_router_lookahead(e_router_lookahead::NO_OP, "", "", {});
```

Comment:
This is rather mysterious without context. Any thoughts on how to make it clearer?

One potential thought is to make router_opts the argument instead of splitting the arguments out, which would make the signature uniform for all lookaheads?

Reply:
Doesn't work because of the segment_inf argument (which is actually not a part of any global data structure).